src/api/services/companyService.ts

```diff
@@ -222,3 +223,50 @@ export function addCalculatedTotalEmissions(companies: any[]) {
       }))
   )
 }
+
+export function addCompanyEmissionChange(companies: any[]) {
```

Comment:
Do we have a type for the companies that we can use here to avoid disregarding type safety?